src/posts/lucky_0.17.cr

```diff
@@ -0,0 +1,41 @@
+class Lucky016Release < BasePost
+  title "Lucky 0.17 has been released!"
+  slug "lucky-0_17-release"
+
+  def published_at
+    Time.utc(year: 2019, month: 8, day: 16)
+  end
+
+  def summary
+    <<-TEXT
+    Lucky v0.17 is a huge release with many enhancements to Avram, Lucky's
+    database library for Postgres.
+    TEXT
+  end
+
+  def content
+    <<-MD
+    Lucky v0.17.0 is now out and has support for the newest Crystal (v0.30.0)!
+
+    ### What's new?
+
+    You can see a [full list of
+    changes](https://github.com/luckyframework/lucky/blob/master/CHANGELOG.md#changes-in-v017),
+    but here are some highlights.
+
+    [Avram](https://github.com/luckyframework/avram) has seen the most changes
+    and improvements.
+
+    * Can now use any column as the primary key. Can also use Int64, UUID or any custom type you implement.
```

Comment:
> Can also use Int64, UUID or any custom type you implement.

You can also use `Int16` and `Int32`. Maybe we can link to the [supported typed](https://www.luckyframework.org/guides/database/migrations#primary-keys) ?

Reply:
Might be good to say something about the default type and also the wording is confusing.

```suggestion
    * Can now use any column as the primary key. The default has changed to `Int64` (from `Int32`). We have a few [supported types](https://www.luckyframework.org/guides/database/migrations#primary-keys) out of the box, but you can also create your own custom types.
```
Not sure where the docs are for creating your own though...

Reply:
Ooo we could link to this: https://dev.to/jwoertink/using-custom-ids-in-lucky-4ibh